src/pages/Lobby.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react'
```

Comment:
Not using this file just yet, but still keeping it around as this is where the meat of the app was working. Will be splitting this out into smaller, easier to manage files